token/bearer.go

```diff
@@ -76,11 +100,15 @@ func (b *BearerToken) SetEACLTable(table *eacl.Table) {
 	b.token.SetBody(body)
 }
 
-func (b BearerToken) EACLTable() *eacl.Table {
-	return eacl.NewTableFromV2(b.token.GetBody().GetEACL())
+// EACLTable returns extended ACL table that should be used during object
+// service request processing with bearer token.
+func (b BearerToken) EACLTable() eacl.Table {
```

Comment:
Lets think how we globally deal with field emptiness? I think there are two approaches:

1. add `bool` value to the return similar to `map` reading
2. define `IsEmpty/IsSet() bool` property on each type

2 is more flexible since we can check emptiness in some context (e.g. verification stage), and require non-empty prop in other contexts (handling stage).

Reply:
I think we have decided to go with option 2. Renambed `Empty()` to `IsEmpty()`.

Reply:
Reverted back to the bool flag after https://github.com/nspcc-dev/neofs-sdk-go/pull/200#discussion_r848450665